packages/frontend/core/src/blocksuite/database-block/properties/created-by/view.tsx

```
@@ -0,0 +1,130 @@
+  textOverflow: 'ellipsis',
+  whiteSpace: 'nowrap',
+  fontSize: '14px',
+  lineHeight: '22px',
+});
+const avatar = css({
+  flexShrink: 0,
+});
+
+const CreatedByCellComponent: ForwardRefRenderFunction<
+  DataViewCellLifeCycle,
+  CellRenderProps<{}, string | null, string | null>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)